site/gatsby-site/blog/incident-report-2022-october/index.fr.mdx

```diff
@@ -1,143 +1,142 @@
 ---
-title: 'AI Incident Roundup – October ‘22'
-metaTitle: 'AI Incident Roundup – October ‘22'
-metaDescription: 'Collection of AI incidents, developments, and ingestion in October of 2022, providing a digestible recap on the latest incidents and reports of the AIID.'
+titre: 'Récapitulatif des incidents d‘IA - 22 octobre'
+metaTitle: 'Récapitulatif des incidents liés à l‘IA - 22 octobre'
+metaDescription: 'Collection d‘incidents, de développements et d‘ingestion d‘IA en octobre 2022, fournissant un récapitulatif digeste des derniers incidents et rapports de l‘AIID.'
 date: '2022-11-14'
 image: './images/aiid-october.webp'
-author: 'Janet Schwartz & Khoa Lam'
+auteur: 'Janet Schwartz & Khoa Lam'
```

Comment:
The keys "titre" and "auteur" should still be in English – it doesn't work otherwise.